airflow/models/dagrun.py

```diff
@@ -83,7 +84,7 @@ class DagRun(Base, LoggingMixin):
     run_id = Column(String(ID_LEN))
     creating_job_id = Column(Integer)
     external_trigger = Column(Boolean, default=True)
-    run_type = Column(String(50), nullable=False)
+    run_type = Column(Enum(DagRunType, native_enum=False, create_constraint=False), nullable=False)
```

Comment:
Does this need a db migration?

Reply:
It doesn't require migration because of `native_enum=False, create_constraint=False`

Reply:
ok, turns out sqlalchmy's enum type uses field name instead of field value as db value, so we ended up getting `SCHEDULED` instead of `scheduled` in queries. I have changed it back to `String` column type for now and overwrite `__str__` method to make it work for mysqlclient.